config/routes.rb

```diff
@@ -99,4 +99,10 @@
 
   root :to => "dashboard#index"
 
+  resources :info do
```

Comment:
These routes are not quite right.
There are many different way to accomplish this .